main.go

```diff
@@ -167,12 +192,20 @@ func (s *Appender) archiveFileName() string {
 	return s.filePath + "_" + ts
 }
 
+// Append inserts line at the end of file and asks file to be rotated if it is too big.
 func (s *Appender) Append(line string) {
```

Comment:
this whole function and many other parts in this file completely ignore errors. i would add error handling if i were you, but that is on you

Reply:
This will be fixed separately.